ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -57,6 +57,10 @@ resources:
       memory: 6Gi
 rpm_build_commands: make rpm-build
 rpm_build_location: _output/rpms/
+raw_steps:
```

Comment:
@stevekuznetsov is it expected NOT to fail on rpm build errors? I guess the target doesn't get even build since nothing consumes it

Reply:
and is this the correct magic to do so?